.changes/unreleased/ENHANCEMENTS-20230925-190924.yaml

```
@@ -0,0 +1,6 @@
+kind: ENHANCEMENTS
```

```suggestion
kind: FEATURES
```

It's a feature since these are new API endpoints (from the perspective of the SDK)